lang/swi-prolog-devel/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           cmake 1.1
 name                swi-prolog-devel
 conflicts           swi-prolog swi-prolog-lite
 epoch               20051223
-version             8.3.19
+version             8.3.20
 revision            1
```

Comment:
if you update the `version` then you should reset the `revision` to `0`

Reply:
Thanks.  Will try to update on next release.

Reply:
that should ideally be done before merging this PR

Reply:
Ah, it is not merged yet.  I pushed a forced update for this PR after rebase to current master and changing 1 to 0.  Hope this works.   Feel free to make minor edits.   Subsequent releases pull the current version and edit the version and hashes using a script, so enhancements are kept.

Reply:
thanks!